src/App.scss

```diff
@@ -1,17 +1,14 @@
 
 :root {
-    --clr-primary-1: hsl(207, 73%, 3%);
-    --clr-primary-2: hsl(210, 61%, 3%);
-    --clr-primary-3: hsl(210, 51%, 3%);
-    --clr-primary-4: hsl(210, 43%, 3%);
-    --clr-primary-5: hsl(204, 36%, 5%);
-
     --clr-black: #222;
     --clr-white: rgb(255, 255, 255);
-    --clr-aHover: rgb(228, 54, 24);
 
+    --clrBlack1: hsl(207, 73%, 3%);
```

Comment:
`--clr-black-pearl` 
converter https://htmlcolors.com/color-converter
this link will help get more name for color https://chir.ag/projects/name-that-color/#02080D